Update some PX4 param references (#2364)
PX4/PX4-user_guide: en/config/safety.md

--- | ---
[NAV_TRAFF_AVOID](../advanced_config/parameter_reference.md#NAV_TRAFF_AVOID) | Set the failsafe action: Disabled, Warn, Return mode, Land mode.

### Quad-chute Failsafe

> Failsafe for when a VTOL vehicle can no longer fly in fixed-wing mode, perhaps because a pusher motor, airspeed sensor or control surface failed.

Reviewer: ```suggestion
Failsafe for when a VTOL vehicle can no longer fly in fixed-wing mode, perhaps because a pusher motor, airspeed sensor, or control surface, failed.
```

> If triggered, the vehicle will immediately switch to hover mode and execute action defined in parameter [COM_QC_ACT](../advanced_config/parameter_reference.md#COM_QC_ACT) .

Reviewer: @sfuhrer I plan to merge all this, but note this change from hover mode to hold mode. I don't know what hover mode is, so if this is actually correct, then let me know and we can revert.
```suggestion
If triggered, the vehicle will immediately switch to Hold mode and execute action defined in parameter [COM_QC_ACT](#COM_QC_ACT) .
```

Author: yeah that hover was correct, Hold is wrong. We need to settle on a consistent naming - I usually refer to the VTOL states to "Hover" (instead of "MC") and "Aerodynamic forward flight" (instead of "FW"). For "Hover" we are quite well aligned in the team, though for how we want to call the fixed-wing/forwrad/cruise/aerodynamic flight we're not aligned. 
Also currently we mostly refer to them as MC and FW modes in the docs, so for now I guess we should keep to that.

Reviewer: So we are absolutely consistent in the docs - Hold mode for position locked in MC and for path locked in FW. In MAVLink the mode is called cruise mode for FW I think, and we should perhaps use that.

Hover would be different, because to me that does not imply "locked in place". But up to you guys to decide and I can make it so. But it has to happen end to end and affect QGC too.

Author: No, for me "Hover" = MC. We also have VTOLs with only one one propeller, so they obviously can never be in "Multicopter" mode. 
For the change here: lets just change it to:
If triggered, the vehicle will immediately switch to **MC** mode and execute action defined in parameter [COM_QC_ACT](#COM_QC_ACT) .

Reviewer: If you can't hold position in your MC I'd call it altitude hold mode: http://docs.px4.io/main/en/flight_modes/altitude_mc.html

I guess the point is what the user will see for the mode in QGC so they don't have a confusing perception.
Calling it MC mode here is fine though.

Can you make this and the other change?

Reviewer: If not, ping me - might get to it before next week. Might now!

:::note
The quad-chute can also be triggered by sending a MAVLINK [MAV_CMD_DO_VTOL_TRANSITION](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_VTOL_TRANSITION) message with `param2` set to `1`.
:::

The parameters that control when the quad-chute will trigger are listed in the table below.

Parameter | Description
--- | ---

> [COM_QC_ACT](../advanced_config/parameter_reference.md#COM_QC_ACT) | Defines automatic action taken after a quad-chute. Can be set to: Warning only, Return, Land, Hold.
> [VT_FW_QC_HMAX](../advanced_config/parameter_reference.md#VT_FW_QC_HMAX) |Maximum height above the ground (if available, otherwise above home if available, otherwise above the local origin) where triggering a quad-chute is possible. At high altitudes there is a big risk to deplete the battery and therefore crash if quad-chuting there.
> [VT_QC_HR_ERROR_I](../advanced_config/parameter_reference.md#VT_QC_HR_ERROR_I) | Quad-chute uncommanded descent threshold. This is the threshold for integrated height rate error to trigger a uncommanded-descent quad-chute. Only checked in altitude-controlled fixed-wing flight. Additional conditions that have to be met for uncommanded descent detection are a positive (climbing) height rate setpoint and a negative (sinking) current height rate estimate.
> [VT_QC_T_ALT_LOSS](../advanced_config/parameter_reference.md#VT_QC_T_ALT_LOSS) | Altitude loss threshold for quad-chute triggering during VTOL transition to fixed-wing
>  flight. Quad-chute triggers the current altitude is more than this value below the altitude at the beginning of the transition.
> [VT_FW_MIN_ALT](../advanced_config/parameter_reference.md#VT_FW_MIN_ALT) | Minimum altitude above Home for fixed-wing flight. When the altitude drops below this value in fixed-wing flight the vehicle a quad-chute is triggered.
> [VT_FW_QC_R](../advanced_config/parameter_reference.md#VT_FW_QC_R) | Absolute roll threshold for quad-chute triggering in FW mode. 
> [VT_FW_QC_P](../advanced_config/parameter_reference.md#VT_FW_QC_P) | Absolute pitch threshold for quad-chute triggering in FW mode.

Reviewer: FYI generally I like to keep context on the page, so I link to the in-page definition rather than the parameters. Hence an anchor for each.

```suggestion
<a id="COM_QC_ACT"></a>[COM_QC_ACT](../advanced_config/parameter_reference.md#COM_QC_ACT) | Defines automatic action taken after a quad-chute. Can be set to: Warning only, Return, Land, Hold.
<a id="VT_FW_QC_HMAX"></a>[VT_FW_QC_HMAX](../advanced_config/parameter_reference.md#VT_FW_QC_HMAX) |Maximum height above the ground (if available, otherwise above home if available, otherwise above the local origin) where triggering a quad-chute is possible. At high altitudes there is a big risk to deplete the battery and therefore crash if quad-chuting there.
<a id="VT_QC_HR_ERROR_I"></a>[VT_QC_HR_ERROR_I](../advanced_config/parameter_reference.md#VT_QC_HR_ERROR_I) | Quad-chute uncommanded descent threshold. This is the threshold for integrated height rate error to trigger a uncommanded-descent quad-chute. Only checked in altitude-controlled fixed-wing flight. Additional conditions that have to be met for uncommanded descent detection are a positive (climbing) height rate setpoint and a negative (sinking) current height rate estimate.
<a id="VT_QC_T_ALT_LOSS"></a>[VT_QC_T_ALT_LOSS](../advanced_config/parameter_reference.md#VT_QC_T_ALT_LOSS) | Altitude loss threshold for quad-chute triggering during VTOL transition to fixed-wing
 flight. Quad-chute triggers the current altitude is more than this value below the altitude at the beginning of the transition.
<a id="VT_FW_MIN_ALT"></a>[VT_FW_MIN_ALT](../advanced_config/parameter_reference.md#VT_FW_MIN_ALT) | Minimum altitude above Home for fixed-wing flight. When the altitude drops below this value in fixed-wing flight the vehicle a quad-chute is triggered.
<a id="VT_FW_QC_R"></a>[VT_FW_QC_R](../advanced_config/parameter_reference.md#VT_FW_QC_R) | Absolute roll threshold for quad-chute triggering in FW mode. 
<a id="VT_FW_QC_P"></a>[VT_FW_QC_P](../advanced_config/parameter_reference.md#VT_FW_QC_P) | Absolute pitch threshold for quad-chute triggering in FW mode.
```
